site_root/input/resources/profiles/StructureDefinition-kfdrc-condition-no-phi.json

```diff
@@ -0,0 +1,55 @@
+{
+    "resourceType": "StructureDefinition",
+    "id": "kfdrc-condition-no-phi",
+    "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-condition-no-phi",
+    "version": "0.1.0",
+    "name": "kfdrc-condition-no-phi",
+    "title": "KFDRC Condition Without PHI",
```

Comment:
Since Meen used `Kids First DRC <thing> without Protected Health Information` can you please change this one so its consistent?